test/package.json

```
@@ -11,7 +11,7 @@
   "dependencies": {
     "apisauce": "^1.1.1",
     "bedrock": "^3.1.1",
-    "bedrock-account": "4.0.0",
+    "bedrock-account": "5.0.0",
```

This is pinning to explicitly `5.0.0` because it lacks the semver which is a carry over from a previous typo.  
```suggestion
    "bedrock-account": "^5.0.0",
```

---

Updated here 3d30c0f